docs/installation/requirements.md

```diff
@@ -27,7 +27,7 @@ Software version requirements are based on a *minimum* requirement.
 | Database | [MariaDB: ^10.2.7](https://mariadb.org/) or [MySQL: ^5.7](https://www.mysql.com/) | `mysql -V` |
 | Caching Service | [Redis](https://redis.io/)  | `redis-server -v` |
 | Service Supervisor | [Supervisor : 3](http://supervisord.org/) | `supervisord -v` |
-| Web Server | nginx, Apache | `nginx -v` |
+| Web Server | proxy server f.e. nginx, trafik or Apache | `nginx -v` |
```

Comment:
Traefik isn't a web server and proxy isn't a requirement. Not quite agree with that PR.

Reply:
well if it is no requirement we can remove the whole row.

However we should probably distinguish about requirements for Docker and requirements for a host installation? If so i agree this PR is not to submit

Reply:
There are no requirements other than Docker and Docker-Composer in Docker installation. It's it already highlighted on the page.